File: WorldFeed/Server/src/Services/Identity/WorldFeed.Identity.API/Auth/RequestAuthUrlInternalParameters.cs
namespace WorldFeed.Identity.API.Auth
{
    using WorldFeed.Common.Public.Parameters.Auth;
    using WorldFeed.Identity.API.Models.Authentication;

    public class RequestAuthUrlInternalParameters : RequestUrlAuthUrlParameters
    {

        public RequestAuthUrlInternalParameters(IRequestAuthUrlParameters parameters, IAuthenticationRequest authRequest) : base(parameters)
        {
            this.AuthRequest = authRequest;
        }

        public IAuthenticationRequest AuthRequest { get; }
    }
}

File: Sharebook/Client/src/app/controllers/Auth/RequestAuthUrlInternalParameters.ts
import {IRequestAuthUrlParameters} from "../../core/Public/Parameters/Auth/IRequestAuthUrlParameters";
import {RequestUrlAuthUrlParameters} from "../../core/Public/Parameters/Auth/RequestUrlAuthUrlParameters";
import {IAuthenticationRequest} from "../../core/Public/Models/Authentication/IAuthenticationRequest";

export class RequestAuthUrlInternalParameters extends RequestUrlAuthUrlParameters {
  constructor(parameters: IRequestAuthUrlParameters, authRequest: IAuthenticationRequest) {
    super(parameters);
    this.authRequest = authRequest;
  }

  public authRequest: IAuthenticationRequest;
}
